README.md

```diff
@@ -103,6 +103,7 @@ Enable the rules that you would like to use.
 * [react/no-danger](docs/rules/no-danger.md): Prevent usage of dangerous JSX properties
 * [react/no-danger-with-children](docs/rules/no-danger-with-children.md): Prevent problem with children and props.dangerouslySetInnerHTML
 * [react/no-deprecated](docs/rules/no-deprecated.md): Prevent usage of deprecated methods
+* [react/no-deprecated-methods](docs/rules/no-deprecated-methods.md): Prevents usage of deprecated component lifecycle methods
```

Comment:
This should be added to `no-deprecated`; since that rule *already* deals with methods, including lifecycle methods.

Reply:
Could please point to an example of life cycle method in `no-deprecated`? I couldn't find any...

Reply:
There may not be an existing example, but the rule still deals with any depreciations - otherwise it’d be called no-deprecated-static or something :-)